.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    ignore:
+      - dependency-name: "org.apache.maven.plugins:maven-compiler-plugin"
```

Comment:
Why should this be ignored?

Reply:
got it from the quarkus splunk extension (https://github.com/quarkiverse/quarkus-logging-splunk/blob/main/.github/dependabot.yml) and thought it could happen that dependabot wants to update transitive dependencies of compiler plugin, but I didn't challenge it. We can delete it and see if dependabot erroneously wants to update transitive dependencies of compiler plugin.

Reply:
Let's try without this and see how it goes. AFAIK, dependabot only deals with direct dependencies and does not check transitive one.